### DISCUSSION TITLE
imagePullSecret not being set in kafka deployment

### DISCUSSION THREAD

**Discussion by sunilkhokalay** [strimzi/strimzi-kafka-operator#6406] [Q&A]
- We use strimzi kafka core version: 2.5.0
- We have set the imagePullSecret under `.spec.kafka.template.pod.imagePullSecrets` of kafka CR.
- The deployment that the operator creates does not have the imagePullSecret in it.

**Comment by scholzj** [comment]
You can probably start by sharing the Strimzi version you use together with the Kafka custom resource you used, the generated Pod YAMLs etc. `.spec.kafka.template.pod.imagePullSecrets` should not affect any Deployments. It should configure the Kafka StatefulSet / Kafka Pods. So no deployments should be affected by it.
